### ISSUE TITLE
`next/getting-started` doesn't work in NPM 6

### ISSUE THREAD

**Issue by wjohnsto** [wpengine/headless-framework#240]
This needs to be investigated. NPM 7 works fine, but there appear to be runtime issues with NPM 6. If necessary we can specify the minimum NPM version in the `package.json`

**Comment by blakewilson** [comment]
This seems to have been fixed with the recent changes and migration in `examples/next/getting-started`.

Was able to successfully build the project locally using Node v12 / NPM v6 as well as deploy to Vercel.

Once #283 is merged into canary this issue should be resolved.
